packages/eslint-plugin/src/rules/no-magic-numbers.ts

```diff
@@ -149,13 +153,34 @@ export default util.createRule<Options, MessageIds>({
       return false;
     }
 
+    /**
+     * Checks if the node parent is a readonly class property
+     * @param node the node to be validated.
+     * @returns true if the node parent is a readonly class property
+     * @private
+     */
+    function isParentTSReadonlyClassProperty(node: TSESTree.Node): boolean {
+      return (
+        typeof node.parent !== 'undefined' &&
```

Comment:
prefer the shorthand

```suggestion
        node.parent &&
```